nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -530,6 +599,17 @@ private String buildCommitMsg(TableMetadata base, TableMetadata metadata, String
     return String.format("Iceberg commit against %s", tableName);
   }
 
+  private String buildCommitMsgForView(ViewMetadata base, ViewMetadata metadata, String viewName) {
```

Comment:
```suggestion
  private String buildCommitMsg(ViewMetadata base, ViewMetadata metadata, String viewName) {
```

Reply:
mainly because the `view` part is already implicit through the method parameters